Rakefile

```diff
@@ -0,0 +1,12 @@
+require 'data_mapper'
+require 'data_mapper_setup'
+
```

Comment:
The rake task isn't working. There is a load error. It can be fixed by changing require 'data_mapper_setup' to require './app/data_mapper_setup'.